Shaders/RHI/VariableRateShading.azsl

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/SrgSemantics.azsli>
+ShaderResourceGroup InstanceSrg : SRG_PerObject
```

Comment:
This is probably be a silly question, but I noticed there are no shaders involved in the [o3de PR](https://github.com/o3de/o3de/pull/13389), doesn't this technique require any shaders in the  engine side?